src/test/java/com/grookage/qtrouper/TrouperTest.java

```diff
@@ -171,4 +171,21 @@ public void trouperStartWithAllEncompassingConfig() throws Exception {
         Assert.assertEquals(20, trouper.getHandlers().size());
         trouper.stop();
     }
+
+    @Test
+    public void priorityQueueConfigTest() throws Exception {
+        val queueConfiguration = QueueConfiguration.builder()
```

Comment:
Don't use lombok val, use java var.

Reply:
Done. 

Added a function to publish message with priority.